templates/root/.subversion/servers

```diff
@@ -0,0 +1,169 @@
+### of 0, i.e. zero, causes a builtin default to be used.
+###
+### Most users will not need to explicitly set the http-library
+### option, but valid values for the option include:
+###    'serf': Serf-based module (Subversion 1.5 - present)
+### Availability of these modules may depend on your specific
+### Subversion distribution.
+###
+### The commented-out examples below are intended only to
+### demonstrate how to use this file; any resemblance to actual
+### servers, living or dead, is entirely coincidental.
+
+### In the 'groups' section, the URL of the repository you're
+### trying to access is matched against the patterns on the right.
+### If a match is found, the server options are taken from the
+### section with the corresponding name on the left.
+
+[groups]
+# group1 = *.collab.net
+# othergroup = repository.blarggitywhoomph.com
+# thirdgroup = *.example.com
+
+### Information for the first group:
+# [group1]
+# http-proxy-host = proxy1.some-domain-name.com
+# http-proxy-port = 80
+# http-proxy-username = blah
+# http-proxy-password = doubleblah
+# http-timeout = 60
+# store-plaintext-passwords = no
+# username = harry
+
+### Information for the second group:
+# [othergroup]
+# http-proxy-host = proxy2.some-domain-name.com
+# http-proxy-port = 9000
+# No username and password for the proxy, so use the defaults below.
+
+### You can set default parameters in the 'global' section.
+### These parameters apply if no corresponding parameter is set in
+### a specifically matched group as shown above.  Thus, if you go
+### through the same proxy server to reach every site on the
+### Internet, you probably just want to put that server's
+### information in the 'global' section and not bother with
+### 'groups' or any other sections.
+###
+### Most people might want to configure password caching
+### parameters here, but you can also configure them per server
+### group (per-group settings override global settings).
+###
+### If you go through a proxy for all but a few sites, you can
+### list those exceptions under 'http-proxy-exceptions'.  This only
+### overrides defaults, not explicitly matched server names.
+###
+### 'ssl-authority-files' is a semicolon-delimited list of files,
+### each pointing to a PEM-encoded Certificate Authority (CA)
+### SSL certificate.  See details above for overriding security
+### due to SSL.
+[global]
+# http-proxy-exceptions = *.exception.com, www.internal-site.org
+# http-proxy-host = defaultproxy.whatever.com
+# http-proxy-port = 7000
+# http-proxy-username = defaultusername
+# http-proxy-password = defaultpassword
+# http-compression = auto
+# No http-timeout, so just use the builtin default.
+# ssl-authority-files = /path/to/CAcert.pem;/path/to/CAcert2.pem
+#
+# Password / passphrase caching parameters:
+# store-passwords = no
+# store-ssl-client-cert-pp = no
+# store-plaintext-passwords = no
+# store-ssl-client-cert-pp-plaintext = no
+store-passwords = yes
```

Comment:
Recommend adding a big, bold, attention-getting comment here pointing out that the change you've made to the defaut `servers` file as created by Subversion is that you've set `store-passwords` and `store-plaintext-passwords` to `yes`.

Reply:
Makes sense

To double down on this line of thought: should I consider just.... deleting all of the comment lines from this file (and the other config files with minorly tweaked defaults)?  That way the only contents are, essentially, this change.

The intent is that this file should never be directly managed anyway (if you change the file on disk then that will get blown away if you ever reprovision)

Reply:
I'm leaning towards this actually, along with a little comment line explaining the context for anybody looking ("if you edit this file, you should edit its template as well")

Reply:
Yup -- totally agree!